bindgen-cli/options.rs

```diff
@@ -204,6 +204,16 @@ where
             Arg::new("no-derive-copy")
                 .long("no-derive-copy")
                 .help("Avoid deriving Copy on any type."),
+            Arg::new("with-derive-custom")
+                .long("with-derive-custom")
+                .help(
+                    "Derive custom traits on any kind of type. \
+                     The derive value must be of the shape <regex>=<kind>=<derive> where <kind> can be one of: \
```

Comment:
I'd be more inclined to keep this as you had it before `--with-derive-custom-struct` iirc and pass `<regex>=<derive>` as arguments.

Reply:
hmm...    ok but I might wait for resolution on the previous comment first